sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1501,11 +1581,11 @@ def task_worker(self):
                     continue
 
                 try:
+                    self.log_notice("Starting CMIS state machine...")
```

Comment:
Do you want to keep this log? If yes, please specify the port name and other relevant information.